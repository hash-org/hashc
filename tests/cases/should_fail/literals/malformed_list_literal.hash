// run=fail, stage=parse

k = [1,
