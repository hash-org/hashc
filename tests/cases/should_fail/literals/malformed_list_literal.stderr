error: encountered unclosed delimiter `[`, add a `]` after the inner expression
 --> $DIR/malformed_list_literal.hash:1:5
1 |   k = [1,
  |       ^ here
